OpenRA.Mods.Common/Traits/Buildings/ProductionAirdrop.cs

```diff
@@ -101,8 +100,8 @@ public override bool Produce(Actor self, ActorInfo producee, string productionTy
 					new FacingInit(spawnFacing)
 				});
 
-				var exitCell = self.Location + exit.ExitCell;
-				actor.QueueActivity(new Land(actor, Target.FromActor(self), WDist.Zero, WVec.Zero, info.Facing, clearCells: new CPos[1] { exitCell }));
+				var landingCells = new CPos[1] { self.Location + info.LandingCell };
+				actor.QueueActivity(new Land(actor, Target.FromActor(self), WDist.Zero, WVec.Zero, info.Facing, clearCells: landingCells));
```

Comment:
Perhaps add a comment? This is reducing flexibility, as now instead of having multiple landing locations you can only have one. The proper solution would be letting the delivery actor reposition himself to the exit which is selected at arrival

Reply:
You are right. This is too invasive. Reverted.